recipes/sentry-native/all/conanfile.py

```diff
@@ -160,7 +163,8 @@ def generate(self):
         tc.variables["SENTRY_TRANSPORT"] = self.options.transport
         tc.variables["SENTRY_PIC"] = self.options.get_safe("fPIC", True)
         tc.variables["SENTRY_INTEGRATION_QT"] = self.options.qt
-        tc.variables["CRASHPAD_WER_ENABLED"] = self.options.wer
+        if self.options.get_safe("wer", False):
+            tc.variables["CRASHPAD_WER_ENABLED"] = True
```

Comment:
You could drop and the if and just to `= get safe` :)

Reply:
I was thinking it was better like this since the variable is not used in the previous versions, but I can change it if it is more standard to set it like that anyway

Reply:
this if fine :)